### ISSUE TITLE
cargo make doc does not compile core

### ISSUE THREAD

**Issue by Orycterope** [sunriseos/SunriseOS#207]
The doc/doc-full rules do not have any dependencies that builds core, and if run in a fresh clone they will fail.

```
error[E0463]: can't find crate for `core`
  |
  = note: the `i386-unknown-none` target may not be installed

error: aborting due to previous error
```

Since there's no "cargo xdoc" command, I don't know how you would go about forcing a core cross-compilation. By building a dummy rust program ?

**Comment by roblabla** [comment]
Could make an issue on cargo-xbuild to add a cargo xdoc command.

**Comment by roblabla** [comment]
On further inspection: There's no reason to set a sysroot for cargo doc. Fixed by #377
